public/app/plugins/datasource/postgres/partials/config.html

```diff
@@ -26,17 +26,40 @@ <h3 class="page-heading">PostgreSQL Connection</h3>
         inputWidth="9"
       />
     </div>
-	</div>
-
-	<div class="gf-form">
-		<label class="gf-form-label width-7">SSL Mode</label>
-		<div class="gf-form-select-wrapper max-width-15 gf-form-select-wrapper--has-help-icon">
-			<select class="gf-form-input" ng-model="ctrl.current.jsonData.sslmode" ng-options="mode for mode in ['disable', 'require', 'verify-ca', 'verify-full']" ng-init="ctrl.current.jsonData.sslmode"></select>
-			<info-popover mode="right-absolute">
-				This option determines whether or with what priority a secure SSL TCP/IP connection will be negotiated with the server.
-			</info-popover>
-		</div>
-	</div>
+  </div>
+  <div class="gf-form">
+    <label class="gf-form-label width-7">SSL Mode</label>
+    <div class="gf-form-select-wrapper max-width-15 gf-form-select-wrapper--has-help-icon">
+      <select class="gf-form-input" ng-model="ctrl.current.jsonData.sslmode" ng-options="mode for mode in ['disable', 'require', 'verify-ca', 'verify-full']" ng-init="ctrl.current.jsonData.sslmode"></select>
+      <info-popover mode="right-absolute">
+        This option determines whether or with what priority a secure SSL (TLS) TCP/IP connection will be negotiated with the server.
+      </info-popover>
+    </div>
+  </div>
+  <div class="gf-form max-width-30" ng-if="ctrl.current.jsonData.sslmode != 'disable'">
+    <span class="gf-form-label width-7">SSL Root Certificate</span>
```

Comment:
Can you please adjust wiht `width-` class name so that the label do not span? not sure what value will be ok, but you can try `width-10` maybe?